app/controllers/user_controller.rb

```diff
@@ -0,0 +1,66 @@
+class UserController < ApplicationController
```

Comment:
do you need this controller ?

Reply:
Removed the user controller (It was there as a junk code)